src/utils/exportUtils/PlainTextExport.ts

```diff
@@ -122,7 +123,11 @@ export default class PlainTextExporter extends Exporter {
             if (this.cancelled) return this.cleanUp();
             if (!haveRendererForEvent(event, false)) continue;
             const textForEvent = await this.plainTextForEvent(event);
-            content += textForEvent && `${new Date(event.getTs()).toLocaleString()} - ${textForEvent}\n`;
+            content +=
+                textForEvent &&
+                `${new Date(event.getTs()).toLocaleString([], {
```

Comment:
Sorry for asking you another change :see_no_evil: We do have some date utils, that we currently use to format the date in other places such as event tiles. I would recommend to use [`formatFullDate`](https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/DateUtils.ts#L95) from there to keep it consistent.

Reply:
although i have made change
but wanted your opinion
so not resolving this thread